docs/user/getting-started.md

```diff
@@ -273,7 +273,7 @@ $ cat Containerfile
 
 FROM quay.io/centos-bootc/centos-bootc:stream9
 
-RUN dnf -y copr enable @redhat-et/flightctl-dev centos-stream-9-x86_64 && \
+RUN dnf -y copr enable @redhat-et/flightctl centos-stream-9-x86_64 && \
```

Comment:
The `centos-stream-9-x86_64` can be removed as we now have the EPEL chroot and so auto-detection works (and we should remove the COS and RHEL chroots I guess).